index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <div id="main">
+    <div id="title">
+      <h1>BOB KAHN</h1>
+    </div>
+    <div id="img-div">
+      <img src="images/220px-Bob_Kahn.jpg" alt="HERO" id="image">
+      <div id="img-caption">
+        Bob Kahn is an American electrical engineer, who, along with Vint Cerf, first proposed the Transmission Control Protocol (TCP) and the Internet Protocol (IP), the fundamental communication protocols at the heart of the Internet.
+      </div>
```

Comment:
Appropriate tag for image captions `figcaption`, using appropriate HTML tags makes your code more readable for SE.